### PR TITLE
Create @gitgraph/js

### PR DESCRIPTION
**Pull request by nicoespeon**
Related to #219.

Contains:
- [x] Set up package
- [x] Create example scenario to test rendering *a la* Storybook
- Implement rendering
  - [x] SVG container
  - [x] Branches paths
  - [x] Commits messages

### CODE REVIEW

File: packages/gitgraph-js/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Now you can use the `<GitGraph>` component:
+
+```js
+const { createGitgraph } = require("@gitgraph/js");
+
+// Get the graph container HTML element.
+const graphContainer = document.getElementById("graph-container");
+
+// Instantiate the graph.
+const gitgraph = createGitgraph(graphContainer);
```
Comment (fabien0102) [inline packages/gitgraph-js/README.md]: Don't forget options :wink: (theme, author…)

Comment (nicoespeon) [reply packages/gitgraph-js/README.md]: Yep, don't worry, I won't 😉

File: packages/gitgraph-react/src/Gitgraph.tsx [hunk 2]
```diff
@@ -296,7 +296,7 @@ class Gitgraph extends React.Component<GitgraphProps, GitgraphState> {
           style={{ font: commit.style.message.font }}
           onClick={commit.onMessageClick}
         >
-          {getMessage(commit)}
+          {commit.message}
```
Comment (nicoespeon) [inline packages/gitgraph-react/src/Gitgraph.tsx]: @fabien0102 

champagne

**Comment by nicoespeon** [comment]
Merging this into master. There is no issue to do that, since it's a new package that is not released yet. It doesn't make `master` less stable, even though this package is still in development (just like `@gitgraph/node`).

**Pull request merged**